src/api/tutorials/create-auth-token.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Create a Sentry Authentication Token
+sidebar_order: 8
+---
+
+To use Sentry's APIs, you must have an authentication token. This tutorial walks you through creating an organizational auth token through an internal integration. Sentry recommends using organizational auth tokens whenever possible, as they aren't linked to specific user accounts.
+
+See our documentation on [authentication](/api/auth/) to learn more about the different types of authentication tokens available.
+
+## Prerequisites
+
+- A Sentry account with an organization-level role of Manager or Admin.
+
+## Create an Internal Integration
+
+[Internal integrations](/product/integrations/integration-platform/internal-integration/) are used to create custom Sentry integrations for your organization. They can also be used to create and manage your organization auth tokens.
+
+1. Open [sentry.io](https://sentry.io/)
+
+1. Click "Settings" in the left menu to open the **Organization Settings** page.
```

Comment:
Why are these all 1?

Reply:
Markdown doesn't care about the number, it automatically renders an ordered list with the correct numbers. https://www.markdownguide.org/basic-syntax/#ordered-lists

I prefer to use all 1s as it's much easier to add/remove steps later.